composer.json

```diff
@@ -51,10 +52,13 @@
         }
     },
 
-    "minimum-stability": "dev",
+    "minimum-stability": "RC",
     "prefer-stable": true,
 
     "extra": {
+        "symfony": {
```

Comment:
Could you please revert this change and in minimum-stability to allow installation of lower symfony versions